docs/addons/amp-addon.md

```diff
@@ -128,6 +128,101 @@ const blueprint = blueprints.EksBlueprint.builder()
   .build(app, 'my-stack-name');
 ```
 
+Pattern # 5 : Configuring rules into AMP. This pattern creates recording rules and/or alerting rules, based on the YAML files provided. A workspace ARN is also required as input.
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import { CfnWorkspace } from 'aws-cdk-lib/aws-aps';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+const ampWorkspaceName = "blueprints-amp-workspace";
+const ampWorkspace = blueprints.getNamedResource(ampWorkspaceName);
+const attrPrometheusEndpoint = ampWorkspace.attrPrometheusEndpoint;
+const ampWorkspaceArn = ampWorkspace.attrArn;
+
+const addOn = new blueprints.addons.AmpAddOn({
+    ampPrometheusEndpoint: attrPrometheusEndpoint,
+    ampRules: {
+        ampWorkspaceArn: ampWorkspaceArn,
+        ruleFilePaths: [
+            __dirname + '/../common/resources/amp-config/alerting-rules.yml',
+            __dirname + '/../common/resources/amp-config/recording-rules.yml'
+        ]
+    }
+})
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .resourceProvider(ampWorkspaceName, new blueprints.CreateAmpProvider(ampWorkspaceName, ampWorkspaceName))
+  .build(app, 'my-stack-name');
+```
+
+The recording rules and alerting rules files should be placed in the same repository that is using this add-on. They should also follow the same format as rules files in standalone Prometheus, like explained in [the AMP User Guide](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-ruler-rulesfile.html):
+
+```yaml
+groups:
+  - name: test
+    rules:
+    - record: metric:recording_rule
+      expr: avg(rate(container_cpu_usage_seconds_total[5m]))
+  - name: alert-test
+    rules:
+    - alert: metric:alerting_rule
+      expr: avg(rate(container_cpu_usage_seconds_total[5m])) > 0
+      for: 2m
+```
+
+An example of rules configuration can be found in the data section of the [EKS monitoring rules file](https://github.com/aws-observability/terraform-aws-observability-accelerator/blob/main/modules/eks-monitoring/rules.tf) in the aws-observability repository.
+
+Pattern # 6 : Configuring the [AWS Distro for OpenTelemetry Collector](https://aws-otel.github.io/docs/getting-started/collector). This pattern enables you to configure the Collector by providing a manifest describing an OpenTelemetryCollector resource:
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: sidecar-for-my-app
+spec:
+  mode: sidecar
+  config: |
+    ...
+    {{javaScrapeSampleLimit}}
+    ...
+    {{javaPrometheusMetricsEndpoint}}
+```
+
+This pattern is useful when you need to customise the configuration of the OpenTelemetryCollector, e.g. to add specific scraping targets and parameters in the Prometheus receiver configuration, for a job capturing metrics of a Java workload.
```

Comment:
minor: customize instead of customise (consistency in the doc)

Reply:
done